### PR TITLE
Update docs for queued imports with middleware

### PR DESCRIPTION
**Pull request by devonmather**

### CODE REVIEW

File: 3.1/imports/queued.md [hunk 1]
```diff
@@ -158,6 +158,39 @@ When dealing with a multi server setup as above, it's possible for the clean up
...
+If you are using Laravel, [job middleware](https://laravel.com/docs/7.x/queues#job-middleware) can be attached to the export class using the `middleware` method.
+
+```php
+namespace App\Exports;
+
+use App\Jobs\Middleware\RateLimited;
+use Maatwebsite\Excel\Concerns\Exportable;
+use Maatwebsite\Excel\Concerns\FromQuery;
+
+class ExportClass implements FromQuery
```
Comment (patrickbrouwers) [inline 3.1/imports/queued.md:173]: I think this should be import?

File: 3.1/imports/queued.md [hunk 2]
```diff
@@ -158,6 +158,39 @@ When dealing with a multi server setup as above, it's possible for the clean up
 ],
 ```
 
+
+## Job Middleware
+
+If you are using Laravel, [job middleware](https://laravel.com/docs/7.x/queues#job-middleware) can be attached to the export class using the `middleware` method.
```
Comment (patrickbrouwers) [inline 3.1/imports/queued.md:164]: export -> import :)

**Comment by devonmather** [comment]
Sorry about the typos!

**Comment by patrickbrouwers** [comment]
No problem @devonmather ! Thanks for updating the docs!

**Pull request merged**